front/app/containers/InitiativesShow/ReactionControl/Status/index.tsx

```diff
@@ -30,9 +30,9 @@ export interface StatusComponentProps {
   initiativeStatus: IInitiativeStatusData;
   initiativeSettings: ProposalsSettings;
   userReacted: boolean;
-  onReaction?: () => void;
-  onCancelReaction?: () => void;
-  onScrollToOfficialFeedback?: () => void;
```

Comment:
They're provided everywhere, so easier if they're required so we don't need to do checks in the code.